server/database/models/lab9/Exercise.js

```diff
@@ -22,6 +22,9 @@ module.exports = (sequelize, DataTypes) => {
         attemptTime: {
           type: DataTypes.DATE,
         },
+        attemptCount: {
+          type: DataTypes.BIGINT,
```

Comment:
Does this need to be a bigint? Maybe we could get away with using integer since this is just a count